airflow/config_templates/config_unit_tests.yml

```diff
@@ -0,0 +1,221 @@
+      example: ~
+      default: "False"
+    load_examples:
+      description: |
+        Whether to load the DAG examples that ship with Airflow. It's good to
+        get started, but you probably want to set this to ``False`` in a production
+        environment
+      version_added: ~
+      type: string
+      example: ~
+      default: "True"
+    fernet_key:
+      description: |
+        Secret key to save connection passwords in the db
+      version_added: ~
+      type: string
+      sensitive: true
+      example: ~
+      default: "{FERNET_KEY}"
+    donot_pickle:
+      description: |
+        Whether to disable pickling dags
+      version_added: ~
+      type: string
+      example: ~
+      default: "False"
+    default_impersonation:
+      description: |
+        If set, tasks without a ``run_as_user`` argument will be run with this user
+        Can be used to de-elevate a sudo user running Airflow when executing tasks
+      version_added: ~
+      type: string
+      example: ~
+      default: ""
+    unit_test_mode:
+      description: |
+        Turn unit test mode on (overwrites many configuration options with test
+        values at runtime)
+      version_added: ~
+      type: string
+      example: ~
+      default: "True"
+database:
+  description: ~
+  options:
+    sql_alchemy_conn:
+      description: |
+        The SqlAlchemy connection string to the metadata database.
+        SqlAlchemy supports many different database engines.
+        More information here:
+        http://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#database-uri
+      version_added: 2.3.0
+      type: string
+      sensitive: true
+      example: ~
+      default: "sqlite:///{AIRFLOW_HOME}/airflow.db"
+    load_default_connections:
+      description: |
+        Whether to load the default connections that ship with Airflow. It's good to
+        get started, but you probably want to set this to ``False`` in a production
+        environment
+      version_added: 2.3.0
+      type: string
+      example: ~
+      default: "True"
+logging:
+  description: ~
+  options:
+    base_log_folder:
+      description: |
+        The folder where airflow should store its log files.
+        This path must be absolute.
+        There are a few existing configurations that assume this is set to the default.
+        If you choose to override this you may need to update the dag_processor_manager_log_location and
+        dag_processor_manager_log_location settings as well.
+      version_added: 2.0.0
+      type: string
+      example: ~
+      default: "{AIRFLOW_HOME}/logs"
+hive:
+  description: ~
+  options:
+    default_hive_mapred_queue:
+      description: |
+        Default mapreduce queue for HiveOperator tasks
+      version_added: ~
+      type: string
+      example: ~
+      default: "airflow"
+smtp:
+  description: |
+    If you want airflow to send emails on retries, failure, and you want to use
```

Comment:
```suggestion
    If you want Airflow to send emails on retries, failure, and you want to use
```

Reply:
Same here. I will not correct those YET. Those will come as separate change